src/stp/new_plays/Attack.cpp

```diff
@@ -38,7 +38,7 @@ Attack::Attack() : Play() {
                                                                                  std::make_unique<role::Defender>(role::Defender("defender_3"))};
 }
 
-uint8_t Attack::score(world_new::World *world) noexcept { return 110; }
+uint8_t Attack::score(world_new::World *world) noexcept { return 80; }
```

Comment:
Why are these constants?... shouldn't it weigh in wehther we have ball or not etc?...

Reply:
The scoring is TBD